### PR TITLE
Decorate links with a target with fa-external-link

### LINKED ISSUE #217: linkify with target should include external link icon in output
We typically include fa-external-link with links that include target="_blank".  For consistency, it seems we would want linkify to behave the same way.  @spadgett, agree/disagree?

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #217

### CODE REVIEW

File: src/ui-services/htmlService.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ angular.module("openshiftCommonUI")
         // Replace any URLs with links.
         return text.replace(/https?:\/\/[A-Za-z0-9._%+-]+\S*[^\s.;,(){}<>"\u201d\u2019]/gm, function(str) {
           if (target) {
-            return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + "</a>";
+            return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + " " + "<i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></a>";
```
Comment (spadgett) [inline src/ui-services/htmlService.js]: nit: You have an extra `+ " " +` that you don't need.

```js
return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + " <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></a>";
```

Comment (rhamilto) [reply src/ui-services/htmlService.js]: My testing indicates we do need the extra space or the icon is butted up against the text.  :/

Comment (spadgett) [reply src/ui-services/htmlService.js]: You can have the space, but still make it one string literal

```
" <i class=\"...\">"
```

Although I wonder if it shouldn't be an `&nbsp;` or `&thinsp;`?

Comment (rhamilto) [reply src/ui-services/htmlService.js]: > You can have the space, but still make it one string literal

That works.  I had tried that before and it didn't work for some reason.  Probably error on my part.  :)

**Comment by rhamilto** [comment]
@spadgett, if you're good with this, it's ready for merge.

**Pull request merged**